content/millennium/r4/financial/account.md

```diff
@@ -0,0 +1,131 @@
+
+### Account Relationships
+
+![Account Relationship Model](/images/account-relationship-model.png)
+
+### Account Types
+
+The account resource supports multiple account types which are defined below. 
+
+* financial-account - Top level patient account
+* guarantor-balance - Self pay balance account.
+* statement - Snapshot in time of a patient statement.
+
+
+The following fields are returned if valued:
+
+* [Account id]( https://hl7.org/fhir/r4/resource-definitions.html#Resource.id ){:target="_blank"} 
+* [Account Identifier](https://hl7.org/fhir/R4/account-definitions.html#Account.identifier){:target="_blank"} 
+* [Status](https://hl7.org/fhir/R4/account-definitions.html#Account.status){:target="_blank"} 
+* [Type](https://hl7.org/fhir/R4/account-definitions.html#Account.type){:target="_blank"}
+* [Subject](https://hl7.org/fhir/R4/account-definitions.html#Account.subject){:target="_blank"} 
+* [Service period](https://hl7.org/fhir/R4/account-definitions.html#Account.servicePeriod){:target="_blank"}
+* [Owner](https://hl7.org/fhir/R4/account-definitions.html#Account.owner){:target="_blank"} 
+* [Guarantor](https:// https://hl7.org/fhir/R4/account-definitions.html#Account.guarantor){:target="_blank"} 
+* [Part of](https://hl7.org/fhir/R4/account-definitions.html#Account.partOf){:target="_blank"}
+* [Extensions including related parts, balance, and state](#extensions){:target="_blank"}
+
+
+## Terminology Bindings
+
+<%= terminology_table(:account, :r4) %>
+
+## Extensions
+
+* [Related Parts]
+* [Balance]
+* [State]
+
+## Search
+
+Search for Accounts that meet supplied query parameters:
+
+    GET /Account?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type          | Description
+--------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------
+ `_id`        | This or any other required search parameter       | [`token`]     | The logical resource id associated with the resource
+ `identifier` | no                                                | [`token`]     | Aliases of the Account like Statement Number
+ `type`       | no                                                | [`token`]     | The specific type of account
+ `patient`    | This and type and identifier, or any other search param, or _id  | [`reference`] | The entity that caused the expenses
+ `-guarantor` | This and type, or any other search param, or _id  | [`reference`] | The parties responsible for balancing the account
+ `_count`     | no                                                | [`number`]    | Paging parameter
+ 
```

Comment:
We should add -offset as a search param

Reply:
I'm pretty sure we had that in there, and then I was told to remove it. Are we sure we should have that as a search param?

Reply:
Talked with Max and the decision is not to have it